website/src/css/styles.scss

```diff
@@ -82,20 +72,17 @@ $spacers: (
     ),
 );
 
-$body-bg: $black;
-$body-color: $white;
+$body-bg: #f9f9f9;
+$body-color: #222233;
```

Comment:
Could you delete the text color override of the cookiebot styles (or remove it altogether) to make sure it uses the same text color as the body? (The override was only there because previously the default body color was white, resulting in white-on-white text)

Reply:
Yes.  This branch will be synced with origin (master).

Reply:
I see no changes in this PR, the color overrides still seem to exist in master:

https://github.com/sourcegraph/about/blob/14ce1719ce81e64cb7572d2097bbbc7fe29e451f/website/src/css/components/_Cookiebot.scss#L5